scripts/ci/gitlab/pipeline/publish.yml

```
@@ -2,32 +2,33 @@
 # This file is part of .gitlab-ci.yml
 # Here are all jobs that are executed during "publish" stage
 
-.build-push-docker-image:
+.build-push-docker-image-common:
   extends:
-    - .build-refs
     - .kubernetes-env
+  stage:                           publish
   variables:
     CI_IMAGE:                      $BUILDAH_IMAGE
     GIT_STRATEGY:                  none
     DOCKERFILE:                    $PRODUCT.Dockerfile
-    IMAGE_NAME:                    docker.io/parity/$PRODUCT
+    IMAGE_NAME:                    docker.io/$IMAGE_PATH
   before_script:
     - cd ./artifacts/$PRODUCT/
     - VERSION="$(cat ./VERSION)"
     - echo "${PRODUCT} version = ${VERSION}"
     - test -z "${VERSION}" && exit 1
   script:
-    - test "$Docker_Hub_User_Parity" -a "$Docker_Hub_Pass_Parity" ||
+    - test "$COMMON_USER" -a "$COMMON_PASS" ||
```

```suggestion
    - test "$DOCKER_USER" -a "$DOCKER_PASS" ||
```
Please change "common" to "docker"

---

done in [37b2167](https://github.com/paritytech/substrate/pull/12675/commits/37b216728781a863dd041a4185f7a8540e309f96) (I used `DOCKERIO_USER`).